test/packages/bad_required_vars/manifest.yml

```diff
@@ -0,0 +1,47 @@
+format_version: 3.1.4
+name: bad_required_vars
```

Comment:
Test packages need to be added to the list of packages in https://github.com/elastic/package-spec/blob/8f1292454250c378af0432606bb99418cc19c753/code/go/pkg/validator/validator_test.go#L28